src/openrct2/management/Research.h

```diff
@@ -17,10 +17,10 @@
 
 struct rct_ride_entry;
 
-enum
+enum class ResearchEntryType : uint8_t
 {
-    RESEARCH_ENTRY_TYPE_SCENERY = 0,
-    RESEARCH_ENTRY_TYPE_RIDE = 1,
+    Scenery = 0,
+    Ride = 1,
 };
```

Comment:
If it's not much to ask, I really think we should start using namespaces more often, I'd do:
```c++
namespace Research
{
    enum class EntryType : uint8_t
    {
        Scenery = 0,
        Ride = 1,
    };
}
```

Usage would be then:
```c++
    Research::EntryType::Scenery
```

Reply:
Yes i was thinking that as well but shouldn't it be Management::Research::EntryType?

Reply:
Oops, I didn't reply fast enough. I think both are fine